pyvista/core/filters/data_set.py

```diff
@@ -2044,8 +2044,8 @@ def elevation(  # type: ignore[misc]
 
     def contour(  # type: ignore[misc]
         self: ConcreteDataSetType,
-        isosurfaces: int = 10,
-        scalars: str | None = None,
+        isosurfaces: int | Sequence[float] = 10,
+        scalars: str | NumpyArray[float] | None = None,
```

Comment:
This looks correct in terms of minimal typing changes. Though, we are making an effort to standardize inputs and accept more general inputs like `VectorLike[float]`. I see this was tried initially but it seems would require modifying the validation logic a bit to accommodate, so we can leave that for a separate PR.